dev.salaniLeo.flake.json

```diff
@@ -0,0 +1,32 @@
+{
+    "app-id" : "dev.salaniLeo.flake",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "43",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "flake",
+    "finish-args" : [
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--filesystem=host"
```

Comment:
this is a broad filesystem permission

Reply:
How do i allow only home files?

Reply:
`home`. but instead you should use portals, which gtk allow to do.

Reply:
Thanks